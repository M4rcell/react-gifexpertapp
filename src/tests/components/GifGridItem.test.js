import React from 'react';
const { shallow } = require("enzyme")
const { GifGridItem } = require("../../components/GifGridItem")

describe('Pruebas con <GifGridItem/>',()=>{
   
    test('Debe de mostrar el componente correctamente', () => {

        const wraper = shallow(<GifGridItem/>)

        expect(wraper).toMatchSnapshot();
        
    })
    

})