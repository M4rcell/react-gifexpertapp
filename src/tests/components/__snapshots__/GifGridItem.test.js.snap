// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con <GifGridItem/> Debe de mostrar el componente correctamente 1`] = `
<div
  className="card"
>
  <img />
  <p />
</div>
`;
